src/components/pdfDocument/PdfDocument.tsx

```diff
@@ -78,6 +88,18 @@ const PdfDocument: FC<IPdfDocumentProps> = (props) => {
     }
   }, [props.activeBboxIndex, bboxMap])
 
+  useEffect(() => {
+    if (activeBbox) {
+      const selectedPage = getSelectedPageByLocation(activeBbox.location);
+      if (selectedPage > -1) {
+        setSelectedPage(selectedPage);
```

Comment:
Please move `setSelectedPage(selectedPage);` out of `if (selectedPage > -1)` scope to set value even if page is not found